src/pages/jj/Main/MainJj.js

```diff
@@ -1,8 +1,63 @@
 import React from 'react';
 
+// #STYLES#
+import './Main.scss';
+import '../../../components/Nav/Nav.scss';
+
+// #COMPONENTS#
+import Nav from '../../../components/Nav/Nav';
+import MainComponent from './MainComponentMap/MainComponent';
+import Recommend from './Aside/Recommend/Recommend';
+import Story from './Aside/Story/Story';
+import Info from './Aside/Info/Info';
+import Account from './Aside/Account/Account';
+
+// APIKEY
+import api from '../config/api';
+
 class MainJj extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      feedList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch(api, {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => this.setState({ feedList: data[0].feeds }));
+  }
```

Comment:
```suggestion
    fetch(api)
      .then(res => res.json())
      .then(([data]) => this.setState({ feedList: data.feeds }));
  }
```
- get method는 디폴트라서 작성해주지 않으셔도 됩니다
- 배열 또한 함수 파라미터 자리에서 직접 구조분해할당 해주실 수 있습니다! 굳이 0번째 인덱스를 적어주는게 어색해보여서요